test/Schemes/BlowupMorphism.jl

```diff
@@ -0,0 +1,14 @@
+R, (x,y,z) = QQ["x", "y", "z"]
```

Comment:
@afkafkafk13 , @simonbrandhorst : Here's the test file, covering the already working functionality. This can be extended by items on a wishlist: What would you like to be able to write? What's the intended behaviour? We can start with the tests and then work our way up to the actual implementation.